server/main.lua

```diff
@@ -49,3 +52,13 @@ end)
 AddEventHandler('playerDropped', function()
     removePlayer(source --[[@as integer]])
 end)
+
+---Lock every spawned vehicle
+---@param vehicle number The entity number of the vehicle.
+AddEventHandler('entityCreated', function (vehicle)
+    if not vehicle or GetEntityType(vehicle) ~= 2 or GetEntityPopulationType(vehicle) > 5 then return end
+    local isDriver = GetPedInVehicleSeat(vehicle, -1) ~= 0
+    local isLocked = (config.lockNPCDrivenCars and isDriver) or (config.lockNPCParkedCars and not isDriver)
+                        and GetVehicleType(vehicle) ~= 'bike' and not getIsVehicleBlacklisted(vehicle)
```

Comment:
We should rename getIsVehicleBlacklisted. I think it's a little unclear what the blacklist means whether it's always locked, or always unlocked so we should just name it the spawnedLocked or spawnedUnlocked list to be more clear.

Reply:
getIsVehicleAlwaysUnlocked?

Reply:
I guess. It's not always unlocked though, it just spawns unlocked?

Reply:
This must also be used when toggle the lock